trailmet/utils/tp_pruning.py

```diff
@@ -22,32 +33,51 @@ def __init__(self, method=None, prune_model=None, org_model=None, batch_size=64,
 
         # for network slimming
         if self.method == "network_slimming":
-            with open(os.path.join(self.root, "resnet50_cifar100.yaml"), 'r') as stream:
+            with open(os.path.join(self.root, "resnet50_cifar100.yaml"), "r") as stream:
```

Comment:
are we hardcoding for resnet50 cifar?

Reply:
In the current state, network slimming only contains one yaml file, because of which I hardcoded it. I can remove the hardcode part if you want.

Reply:
I have removed the hardcode part now.